package/gluon-xlat464-plat/files/xlat464plat.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/sh
+# Copyright 2018 Vincent Wiemann <vincent.wiemann@ironai.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2
+# as published by the Free Software Foundation
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+CTR=/proc/net/nat46/control
+
+
+[ -n "$INCLUDE_ONLY" ] || {
+  . /lib/functions.sh
+  . ../netifd-proto.sh
+  init_proto "$@"
+}
+
+
+proto_xlat464plat_init_config() {
+  proto_config_add_bool "debug"
+  proto_config_add_string "local_v4"
+  proto_config_add_string "local_v6"
+  proto_config_add_string "nat66addr"
+  proto_config_add_string "tunlink"
+  proto_config_add_string "zone"
+  proto_config_add_string "zone6"
+  available=1
+  no_device=1
+}
+
+
+proto_xlat464plat_setup() {
+  local config="$1"
+
+  local plat_cfg="config ${config}"
+  local debug
+  local local_v4
+  local local_v6
+  local nat66addr
+  local tunlink
+  local zone
+  local zone6
+
+  config_load network
+  config_get_bool	debug		"${config}" "debug" 0
+  config_get		local_v4	"${config}" "local_v4"
+  config_get		local_v6	"${config}" "local_v6"
+  config_get		nat66addr	"${config}" "nat66addr"
+  config_get		tunlink		"${config}" "tunlink"
+  config_get		zone		"${config}" "zone"
+  config_get		zone6		"${config}" "zone6"
+
+  nat66addr="${nat66addr:-fddd:c:1a4c:1a4c:1a4c:1a4c:c000:8}"
+  local_v4="${local_v4:-0.0.0.0/0}"
+  local_v6="${local_v6:-64:ff9b::/96}"
+  tunlink="${tunlink:-wan}"
+  zone="${zone:-wan}"
+  zone6="${zone6:-lan}"
+
+
+  ( proto_add_host_dependency "$config" "${local_v4%%/*}" "${tunlink}" )
+
+  if [ ${debug} -ne 0 ]; then
+    plat_cfg="${plat_cfg} debug 1"
+  fi
+
+  plat_cfg="${plat_cfg} local.style RFC6052 local.v6 ${local_v6} local.v4 ${local_v4}"
+  plat_cfg="${plat_cfg} remote.style NONE remote.v4 192.0.0.8/32 remote.v6 ${nat66addr}/128"
+
+  echo "add ${config}" > ${CTR}
+  echo "${plat_cfg}" > ${CTR}
+
+
+  proto_init_update "${config}" 1
+
+  proto_add_ipv6_address "${local_v6%%:/*}c000:1" "128"
+  proto_add_ipv6_route "${local_v6%%/*}" "${local_v6##*/}"
+  proto_add_ipv4_route "192.0.0.8" "32"
+
+	proto_add_data
```

Comment:
suddenly the indentation changes? please check the complete file to have an indentation matching other shell scripts in Gluon